src/dune_engine/opam_file.mli

```diff
@@ -7,7 +7,9 @@ open OpamParserTypes
 type t = opamfile
 
 (** Load a file *)
-val load : Path.t -> t
+val load : Path.t -> (t, exn) result Memo.t
+
+val load_untracked : Path.t -> t
```

Comment:
Shall we keep the convention of using `Untracked` modules?

Also, the type suggests that it's not only untracked but can't fail. Is that true?

Reply:
Tbh, I'm not a fan of the Untracked convention here because the return type already provides this information. I did the following: inline the IO functions in the callers. It makes both the error handling and tracked vs. untracked more clear.